### PR TITLE
Fixes for new stack

### PR DESCRIPTION
**Pull request by piotrze**
https://jenkins.sandbox.near-me.com/job/nm-org/job/marketplace-nearme-example/job/fixes-for-new-stack/13/console

I think we should keep two branches as we need also test old stack. So might be good idea to branch RC before merging.

### CODE REVIEW

File: marketplace_builder/custom_model_types/application_files.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 name: application_files
-instance_profile_types:
-  - developer
 custom_attributes:
```
Comment (Slashek) [inline marketplace_builder/custom_model_types/application_files.yml]: rename all `custom_attributes:` to `properties`

File: marketplace_builder/custom_model_types/application_files.yml [hunk 2]
```diff
@@ -1,6 +1,4 @@
...
   - name: application_file
     attribute_type: file
```
Comment (Slashek) [inline marketplace_builder/custom_model_types/application_files.yml]: rename all `attribute_type` to `type`

File: marketplace_builder/form_configurations/contact_edit_form.liquid [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+name: contact_edit_form
+resource: Customization
```
Comment (Slashek) [inline marketplace_builder/form_configurations/contact_edit_form.liquid]: use the actual custom model type name please

File: marketplace_builder/form_configurations/contact_edit_form.liquid [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+resource_owner: anyone
+redirect_to: /contacts
+configuration:
```
Comment (Slashek) [inline marketplace_builder/form_configurations/contact_edit_form.liquid]: rename `configuration:` to `fields:`

**Comment by piotrze** [comment]
@Slashek Applied new names.

**Pull request merged**